src/components/Header/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { CustomAvatar } from 'components/shared';
+import React from 'react';
+import avatar1 from 'assets/img/avatar1.svg';
+import avatar2 from 'assets/img/avatar2.svg';
+import avatar3 from 'assets/img/avatar3.svg';
+import avatar4 from 'assets/img/avatar4.svg';
+import header1 from 'assets/img/header-img.svg';
+import styles from './style.module.scss';
+
+export const HeaderComponent = () => (
+	<div>
+		<h1 className={styles.app__title}>
```

Comment:
as I said previosly don't use app__ 
at first, it's not an app component. just title classname. change all

Reply:
rename all classes